src/powershell/devcontainer-feature.json

```diff
@@ -14,8 +14,13 @@
             ],
             "default": "latest",
             "description": "Select or enter a version of PowerShell."
-        }
-    },
+        },
```

Comment:
Can we bump the patch version in here? https://github.com/devcontainers/features/blob/262a7424215b67d89f1ab26483ac4cad90d87dcf/src/powershell/devcontainer-feature.json#L3

Reply:
Done!